store/snapshots/store.go

```diff
@@ -172,7 +172,6 @@ func (s *Store) Load(height uint64, format uint32) (*types.Snapshot, <-chan io.R
 				_ = pw.CloseWithError(err)
 				return
 			}
-			defer chunk.Close()
```

Comment:
If you want to avoid the loop defers, please create a closure down below as I showed above.